postgres/statefulset.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: _-postgres
```

Comment:
Conversely: Should the sta name be `postgres` or `appname-postgres`?

I leaned towards `appname-postgres` here because it is easier to read in the pod list.